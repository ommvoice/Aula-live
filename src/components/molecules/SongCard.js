import React from 'react';

const SongCard = ({imgsrc ,text }) => {
	return (
		<div className="col-md-4">
			<div className="song-card margin-bottom">
				<img src={imgsrc} />
				<div className="container">
					<div className="text">{text}</div>
				</div>
			</div>
		</div>
	);
}

export default SongCard;