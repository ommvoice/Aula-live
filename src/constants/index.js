export const API = 'https://rss.itunes.apple.com';
export const PROXY = 'https://cors-anywhere.herokuapp.com/';
export const SONG_CATEGORIES = [
  { title: 'Top Albums', value: 'top-albums' },
	{ title: 'Top Songs', value: 'top-songs' },
	{ title: 'Hot Tracks', value: 'hot-tracks' },
	{ title: 'New Releases', value: 'new-releases' },
	{ title: 'Coming Soon', value: 'coming-soon' }
];
