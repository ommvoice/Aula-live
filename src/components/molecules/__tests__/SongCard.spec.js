import React from 'react';
import { shallow } from 'enzyme';
import SongCard from '../SongCard';
import { SONG_CATEGORIES } from '../../../constants';

describe('SongCard', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SongCard text="test" />);

    expect(wrapper).toMatchSnapshot();
  });
});
