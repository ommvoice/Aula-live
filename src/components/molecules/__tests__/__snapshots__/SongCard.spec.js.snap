// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongCard renders correctly 1`] = `
<div
  className="col-md-4"
>
  <div
    className="song-card margin-bottom"
  >
    <img />
    <div
      className="container"
    >
      <div
        className="text"
      >
        test
      </div>
    </div>
  </div>
</div>
`;
